src/pages/test-results-search/test-results-search.ts

```diff
@@ -101,4 +102,11 @@ export class TestResultsSearchPage extends BasePageComponent {
       .subscribe();
   }
 
+  myHeaderFn(record: any, recrodIndex: any): string {
```

Comment:
typo here

Reply:
What's this method for btw?

Reply:
The `ion-list` needs some function to determine which items in the list need header. In our case just the first item will need header, but we purely use this only for design so we don't need to return any information just an empty string.

Corrected the typo.